terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/test/java/org/terasoluna/gfw/common/codelist/i18n/SimpleI18nCodeListTest.java

```diff
@@ -157,6 +317,72 @@ public void testSetColumnss02() {
+                                                                   // columns
+
+        Map<String, String> row1 = testSetColumns01.asMap(Locale.US);
+        assertThat(row1, is(notNullValue()));
+        assertThat(row1.keySet().toString(), is("[0, 1, 2, 3, 4, 5, 6]")); // check
+                                                                           // order
+        assertThat(row1.get("0"), is("Sun."));
+        assertThat(row1.get("1"), is("Mon."));
+        assertThat(row1.get("2"), is("Tue."));
+        assertThat(row1.get("3"), is("Wed."));
+        assertThat(row1.get("4"), is("Thu."));
+        assertThat(row1.get("5"), is("Fri."));
+        assertThat(row1.get("6"), is("Sat."));
+
+        Map<String, String> row2 = testSetColumns01.asMap(Locale.JAPAN);
+        assertThat(row2, is(notNullValue()));
+        assertThat(row2.keySet().toString(), is("[0, 1, 2, 3, 4, 5, 6]")); // check
+                                                                           // order
+        assertThat(row2.get("0"), is("日"));
+        assertThat(row2.get("1"), is("月"));
+        assertThat(row2.get("2"), is("火"));
+        assertThat(row2.get("3"), is("水"));
+        assertThat(row2.get("4"), is("木"));
+        assertThat(row2.get("5"), is("金"));
+        assertThat(row2.get("6"), is("土"));
+    }
+
+    @Test
+    public void testSetColumns04() {
+        assertThat(testSetColumns01.codeListTable.size(), is(14)); // 2 rows x 7
+                                                                   // columns
+
+        Map<String, String> row = testSetColumns01.asMap(Locale.CHINESE);
+        assertThat(row, is(notNullValue()));
+        assertThat(row.keySet().toString(), is("[0, 1, 2, 3, 4, 5, 6]")); // check
+                                                                          // order
+        assertThat(row.get("0"), is("Sun."));
+        assertThat(row.get("1"), is("Mon."));
+        assertThat(row.get("2"), is("Tue."));
+        assertThat(row.get("3"), is("Wed."));
+        assertThat(row.get("4"), is("Thu."));
+        assertThat(row.get("5"), is("Fri."));
+        assertThat(row.get("6"), is("Sat."));
+    }
+
+    @Test
+    public void testSetColumns05() {
+        assertThat(testSetColumns02.codeListTable.size(), is(7)); // 1 rows x 7
+                                                                  // columns
+
+        Map<String, String> row = testSetColumns02.asMap(Locale.CHINESE);
+        assertThat(row, is(notNullValue()));
+        assertThat(row.keySet().toString(), is("[0, 1, 2, 3, 4, 5, 6]")); // check
+                                                                          // order
+        assertThat(row.get("0"), is("Sun."));
+        assertThat(row.get("1"), is("Mon."));
+        assertThat(row.get("2"), is("Tue."));
+        assertThat(row.get("3"), is("Wed."));
+        assertThat(row.get("4"), is("Thu."));
+        assertThat(row.get("5"), is("Fri."));
+        assertThat(row.get("6"), is("Sat."));
+    }
+
```

Comment:
- [ ] It is hard to understand which is the test focusing on locale resolution, and I do not know which pattern is covered, which test should be changed if the pattern is changed.
It would be better to restructure the test patterns.

Reply:
Modified by 31312fc
The whole was fixed.
Especially the method name is roughly changed.

Reply:
Exclude underscore from method name according to Codacy's indicatio.
5c987140379120e6d2c02de85aa903bb4e6587d1